config/routes.rb

```diff
@@ -20,11 +20,19 @@
     concerns :exportable
   end
 
-  resources :bookmarks do
-    concerns :exportable
+  constraints(id: /((?!\/)|(?!articles).)*/) do # EDS identifier rules (e.g., db__acid) where acid has all sorts of different punctuation
```

Comment:
Struggled w/ this a bit.  It kept interpreting `articles` as an identifier in bookmarks.